Add AV1002: Only pass things to a constructor that most or all members need (#309)
dennisdoomen/CSharpGuidelines: _rules/1002.md

---
rule_id: 1002
rule_category: class-design

> title: Only pass things to a constructor that most or all members need

Reviewer: ```suggestion
title: Only include parameters in a constructor that most or all members need
```

This is about the constructor's definition, not about callers that pass information.
